gradle.properties

```diff
@@ -1,9 +1,17 @@
-ideaVersion=IC-2018.3
+ideaVersion=IC-2019.3
```

Comment:
Seems to me that your lowest version is now 2019.3 so IJ.yml does not match

Reply:
IJ.yml was corrected, see here: https://github.com/redhat-developer/intellij-kubernetes/pull/245/files#diff-d2a86296b50b44799992fba964e1e33c7d03d00535fd5e24388d3981f6415332L17

Reply:
It's not aligned because IJ.yml used 2019.1 as the lowest

Reply:
sorry, my bad. Will be corrected in #243 and this one rebased.